app/views/users/_sidebar.html.erb

```diff
@@ -44,7 +44,7 @@
           </div>
         <% end %>
       <% end %>
-      <% cache "user-profile-sidebar-stats-#{@user.id}-#{@user.last_article_at}-#{@user.last_comment_at}-#{@user.last_followed_at}", expires_in: 10.days do %>
+      <% cache "user-profile-sidebar-stats-#{@user.id}-#{@user.last_article_at}-#{@user.last_comment_at}-#{@user.following_tags_count}-#{@user.last_followed_at}", expires_in: 10.days do %>
```

Comment:
```suggestion
      <% cache "user-profile-sidebar-stats-#{@user.id}-#{@user.last_article_at&.rfc3339}-#{@user.last_comment_at&.rfc3339}-#{@user.following_tags_count}-#{@user.last_followed_at&.rfc3339}", expires_in: 10.days do %>
```

While we are here could we format these timestamps a bit nicer for the cache key? Using rfc3339 removes all of the spaces which can make searching for the keys easier. 
```ruby
[41] pry(main)> User.last.created_at.rfc3339
=> "2020-10-19T18:13:01Z"
[43] pry(main)> User.last.created_at.to_s
=> "2020-10-19 18:13:01 UTC"
```